tests/integration/targets/lambda_alias/defaults/main.yml

```diff
@@ -1,10 +1,7 @@
 ---
 # defaults file for lambda integration test
-lambda_function_name: '{{ resource_prefix }}'
 # IAM role names have to be less than 64 characters
-# The 8 digit identifier at the end of resource_prefix helps determine during
-# which test something was created and allows tests to be run in parallel
-# Shippable resource_prefixes are in the format shippable-123456-123, so in those cases
-# we need both sets of digits to keep the resource name unique
-unique_id: "{{ resource_prefix | regex_search('(\\d+-?)(\\d+)$') }}"
+# we hash the resource_prefix to get a shorter, unique string
+unique_id: "{{ resource_prefix | hash('md5') }}"
+lambda_function_name: '{{ unique_id }}'
```

Comment:
Slight concern here: this is going to result in a load of lambda resources with just an MD5 has as the ID, and nothing to tie them back to ansible tests.  While this is fine for Ansible's CI, this may cause confusion in private test environments.